blueprints/app/files/testem.js

```diff
@@ -1,3 +1,5 @@
+const { isCI } = require('ci-info');
```

Comment:
Should we add a `'use strict';` up here now that this file is getting more complicated?

Reply:
Ya, good idea